scripts/deploy/github/deploy-kfp.sh

```diff
@@ -32,7 +32,7 @@ then
   exit $EXIT_CODE
 fi
 
-PLATFORM_AGNOSTIC_MANIFESTS="manifests/kustomize/env/platform-agnostic"
+PLATFORM_AGNOSTIC_MANIFESTS="scripts/deploy/github/manifests/test"
```

Comment:
Can we rename `PLATFORM_AGNOSTIC_MANIFESTS` to `TEST_MANIFESTS`?

Reply:
Ah yes, good point, will do!